packages/styled-components/src/sheet/test/Rehydration.test.js

```diff
@@ -23,31 +23,37 @@ describe('outputSheet', () => {
 
     const output = outputSheet(sheet)
       .trim()
-      .split('\n');
-
-    expect(output).toEqual([
-      '.a {}',
-      `${SC_ATTR}.g11[id="idA"]{content:"nameA,"}`,
-      '.b {}',
-      `${SC_ATTR}.g22[id="idB"]{content:"nameB,"}`,
-    ]);
+      .split('/*!sc*/');
+
+    expect(output).toMatchInlineSnapshot(`
+      Array [
+        ".a {}",
+        "
+      data-styled.g11[id=\\"idA\\"]{content:\\"nameA,\\"}",
```

Comment:
`data-styled` should probably still be `${SC_ATTR}` here, to accommodate the different `process.env` values it could be (https://github.com/styled-components/styled-components/blob/d0e8eab8cf391630294c6ac9e81f206eb64e5e53/packages/styled-components/src/constants.js#L6:14)